<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Final Project</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QWidget" name="vertCanvas" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>591</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="HDRcheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HDR</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ExposureCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="UseGammacheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BloomCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bloom</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ExposureTextbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>30</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AddMetaballButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add ball</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetMetaballsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DofCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DOF</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ReflectionsCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflections</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="ballsGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>105</height>
     </size>
    </property>
    <property name="title">
     <string>Metaball Attributes</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QSlider" name="ballSlider1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="ballSlider1Edit">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="attQuadraticLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Field Radius</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sphereLBEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="sphereLREdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="sphereLGEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ball1ColorButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>86</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sphereLLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>62</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scene Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddMetaballButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addMetaballButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ResetMetaballsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeMetaballButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>addMetaballButtonClicked()</slot>
  <slot>removeMetaballButtonClicked()</slot>
 </slots>
</ui>
